### ISSUE TITLE
Plugin not working with Gradle version above 4.7

### ISSUE THREAD

**Issue by razmaniandevil** [typelead/gradle-eta#27]
Applying the gradle plugin (0.7.7) causes the following error :

```
org.gradle.api.internal.DefaultDomainObjectCollection.<init>(Ljava/lang/Class;Ljava/util/Collection;)V
```

**Comment by rahulmutt** [comment]
@razmaniandevil We use a couple of internal Gradle API, so I guess in 4.8+ they prevented plugins from accessing it. Thanks for reporting!

**Comment by rahulmutt** [comment]
It turns out we had no proper reason for using that internal API. We removed its usage and the build with 4.8+ succeeds now. Thanks again!

Fixed in v0.7.8 which is published.

**Comment by razmaniandevil** [comment]
Awesome ! Thanks !
